test/sed.js

```diff
@@ -74,4 +74,24 @@ var result = shell.sed('l*\\.js', '', 'resources/grep/file');
 assert.ok(!shell.error());
 assert.equal(result, 'alphaaaaaaabeta\nhowareyou\nalphbeta\nthis line ends in\n\n');
 
+shell.cp('-f', 'resources/file1', 'tmp/file1');
+shell.cp('-f', 'resources/file2', 'tmp/file2');
+
+// multiple file names
+var result = shell.sed('test', 'hello', 'tmp/file1', 'tmp/file2');
+assert.equal(shell.error(), null);
+assert.equal(result, 'hello1\nhello2');
+
+// array of file names
+var result = shell.sed('test', 'hello', ['tmp/file1', 'tmp/file2']);
+assert.equal(shell.error(), null);
+assert.equal(result, 'hello1\nhello2');
+
+// multiple file names, with in-place-replacement
+var result = shell.sed('-i', 'test', 'hello', ['tmp/file1', 'tmp/file2']);
+assert.equal(shell.error(), null);
+assert.equal(result, 'hello1\nhello2');
+assert.equal(shell.cat('tmp/file1'), 'hello1');
+assert.equal(shell.cat('tmp/file2'), 'hello2');
```

Comment:
I modified one of these tests to use the regex `/t.*st/` instead of `'test'`. It shouldn't change anything, since this doesn't modify how regexes are handled.